protocol/common/raw_message.go

```diff
@@ -14,16 +14,46 @@ const (
 	KeyExMsgRekey CommKeyExMsgType = 2
 )
 
+// ResendType There are distinct mechanisms for retrying send messages: Datasync supports only direct messages (1-to-1 or private group chats)
+// because it requires an acknowledgment (ACK). As implemented, sending a message to a community, where hundreds of
+// people receive it, would lead all recipients to attempt sending an ACK, resulting in an excessive number of messages.
+// Datasync utilizes ACKs, but community messages do not, to avoid this issue. However, we still aim to retry sending
+// community messages if they fail to send or if we are offline.
+type ResendType uint8
+
+const (
+	// ResendTypeNone won't resend
+	ResendTypeNone ResendType = 0
+	// ResendTypeDataSync use DataSync which use MVDS as underlying dependency to resend messages
```

Comment:
As in other cases, maybe it makes sense to add a link to relevant code doing the resend.

Reply:
Thanks!